src/environments/environment.test.ts

```diff
@@ -4,6 +4,6 @@ export const environment = {
   couchAddress: 'http://127.0.0.1:5984/',
   centerAddress: 'earth.ole.org:2200',
   centerProtocol: 'https',
-  parentProtocol: 'https',
+  parentProtocol: 'planet-parent-protocol',
```

Comment:
@lmmrssa This is not overwritten by the docker script.  That only runs for the production version.  Please do not change as this will break tests.

Reply:
thought so :) changes reverted